### PR TITLE
Fund and release

### PR DESCRIPTION
**Pull request by cryptoAtwill**
Fund and release gateway actor call.

### CODE REVIEW

File: src/manager/lotus.rs [hunk 1]
```diff
@@ -146,12 +146,45 @@ impl<T: JsonRpcClient + Send + Sync> SubnetManager for LotusSubnetManager<T> {
...
+        );
+        message.value = amount;
+
+        self.mpool_push_and_wait(message).await?;
+        Ok(())
     }
 
-    async fn release(&self, _subnet: SubnetID, _from: Address) -> Result<()> {
-        todo!()
+    async fn release(&self, subnet: SubnetID, from: Address) -> Result<()> {
```
Comment (adlrocha) [inline src/manager/lotus.rs:171]:
```suggestion
    async fn release(&self, subnet: SubnetID, from: Address, amount: TokenAmount) -> Result<()> {
```

File: src/manager/lotus.rs [hunk 2]
```diff
@@ -146,12 +146,45 @@ impl<T: JsonRpcClient + Send + Sync> SubnetManager for LotusSubnetManager<T> {
...
+            ));
+        }
+
+        let message = MpoolPushMessage::new(
+            Address::new_id(DEFAULT_IPC_GATEWAY_ADDR),
+            from,
+            ipc_gateway::Method::Release as MethodNum,
+            vec![],
+        );
+
```
Comment (adlrocha) [inline src/manager/lotus.rs:185]: Release also expects a token amount to release from the subnet.
```suggestion
        message.value = amount;
```

**Review by adlrocha — APPROVED** [review]
LGTM, thanks. Fix the issue with release, that expects a token amount also and feel free to merge.

File: src/manager/lotus.rs (hunk 1 above)
Comment (cryptoAtwill) [reply src/manager/lotus.rs]: updated and resolving this

File: src/manager/lotus.rs (hunk 2 above)
Comment (cryptoAtwill) [reply src/manager/lotus.rs:186]: updated and resolving this

**Pull request merged**